src/sage/rings/dirichlet_series_ring.py

```diff
@@ -0,0 +1,153 @@
+r"""
```

Comment:
The new modules should be added to the documentation index

Reply:
Which file(s) do I need to touch to do this? I couldn't find this in the developer's guide.

Reply:
@kedlaya I would put it in `src/doc/en/reference/power_series/index.rst`.